extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-configuration.js

```diff
@@ -65,11 +65,9 @@ export class QwcConfiguration extends LitElement {
       }
       
       .lock-icon {
-        color: var(--lumo-error-color-50pct);
```

Comment:
I would keep color in here, but just make it another color. Black is too "in your face" How about `--lumo-contrast-60pct` (see https://vaadin.com/docs/v23/styling/lumo/design-tokens/color)

Reply:
I like it, it matches the `Filter` icon color too. I'll update it 

Night mode: 
![image](https://user-images.githubusercontent.com/54133/228714019-80487a29-f244-460c-a271-6a6e8815e6f6.png)

Day mode:
![image](https://user-images.githubusercontent.com/54133/228714100-658fc6cd-543b-4414-805b-079d981755f0.png)

Reply:
Nice!